src/include/simeng/Core.hh

```diff
@@ -31,12 +66,16 @@ class Core {
   virtual const ArchitecturalRegisterFileSet& getArchitecturalRegisterFileSet()
       const = 0;
 
+  /** Send a syscall to the system's syscall handler. */
+  virtual void sendSyscall(OS::SyscallInfo) const = 0;
+
+  /** Communicate the result of a syscall to the core's active exception
```

Comment:
Can the core have multiple exception handlers? what does "active exception handler" signify? 
Also would be better to reword it as "This method receives the result of an initiated syscall and communicates the result to the core's....  "

Reply:
I've just reworded it to your suggestion as it was good. The comment was an artefact of when we have multiple instances of the exception handler